examples/arduino/z_get.ino

```diff
@@ -21,16 +21,12 @@
 #define SSID "SSID"
 #define PASS "PASS"
 
-#define CLIENT_OR_PEER 0  // 0: Client mode; 1: Peer mode
-#if CLIENT_OR_PEER == 0
```

Comment:
What's the reason for not using the CLIENT_OR_PEER approach?

Reply:
Basically the #if was ignored and compiler complained that #mode and #connect were redefined.